### PR TITLE
feat(APPT-STQA-74): Added the appointment booking web page (layout)

### PR DESCRIPTION
**Pull request by Colorin09**
Here is the layout for the book appointment webpage. the database data is not done yet.

### CODE REVIEW

**Comment by sylveon** [comment]
Lots of clutter going on here, could you fix that?
image

File: src/main/java/org/springframework/samples/petclinic/web/AppointmentController.java [hunk 1]
```diff
@@ -1,17 +1,31 @@
...
+import org.springframework.samples.petclinic.service.ClinicService;
+
 import org.springframework.web.bind.annotation.GetMapping;
 
 import java.util.Map;
 
 @Controller
 public class AppointmentController {
 
+    private static final String appointForm = "appointments/appointments";
```
Comment (sylveon) [inline src/main/java/org/springframework/samples/petclinic/web/AppointmentController.java]: Use shout case for constants

File: src/main/java/org/springframework/samples/petclinic/web/AppointmentController.java [hunk 2]
```diff
@@ -1,17 +1,31 @@
...
+    public AppointmentController(ClinicService clinicService) {
+        this.clinicService = clinicService;
+    }
+    
+    /*Controller for the booking appointment page*/
     @GetMapping(value = "/appointments")
-    public String initFindForm(Map<String, Object> model) {
-        model.put("pet", new Pet());
-        return "appointments/appointments";
+    public String initCreationForm(Map<String, Object> vetInfo) {
```
Comment (sylveon) [inline src/main/java/org/springframework/samples/petclinic/web/AppointmentController.java:24]: Unused parameter, remove it

File: src/main/java/org/springframework/samples/petclinic/web/AppointmentController.java [hunk 3]
```diff
@@ -1,17 +1,31 @@
...
+    private static final String appointForm = "appointments/appointments";
+    private final ClinicService clinicService;
+
+    @Autowired
+    public AppointmentController(ClinicService clinicService) {
+        this.clinicService = clinicService;
+    }
+    
+    /*Controller for the booking appointment page*/
     @GetMapping(value = "/appointments")
```
Comment (sylveon) [inline src/main/java/org/springframework/samples/petclinic/web/AppointmentController.java]:
```suggestion
    @GetMapping(value = "/appointments/create")
```

File: src/main/webapp/WEB-INF/jsp/appointment.jsp [hunk 4]
```diff
@@ -0,0 +1,56 @@
+<%@ page session="false" trimDirectiveWhitespaces="true" %>
```
Comment (sylveon) [inline src/main/webapp/WEB-INF/jsp/appointment.jsp]: Delete this file

File: src/main/webapp/WEB-INF/jsp/appointments/appointments.jsp [hunk 5]
```diff
@@ -1,14 +1,56 @@
 <%@ page session="false" trimDirectiveWhitespaces="true" %>
```
Comment (sylveon) [inline src/main/webapp/WEB-INF/jsp/appointments/appointments.jsp]: Rename this to createAppointment.jsp

**Review by sylveon — CHANGES_REQUESTED** [review]
Looks pretty decent, few minor comments though

**Review by sylveon — DISMISSED** [review]

**Review by GSwizzy — DISMISSED** [review]

**Review by sylveon — APPROVED** [review]

**Review by GSwizzy — APPROVED** [review]

**Pull request merged**